xml_inscriptions/transcribed/NJ.Princ.PU.AM.2003-130.xml

```diff
@@ -0,0 +1,170 @@
+                            <supportDesc ana="#stone">
+                                <support>
+                                    <p> </p>
+                                    <dimensions type="surface" unit="cm">
+                                        <height>1.5</height>
+                                        <width>13.5</width>
+                                        <depth>2.5</depth>
+                                    </dimensions>
+                                </support>
+                                <condition ana="#fragment.single">
+                                    <p/>
+                                </condition>
+                            </supportDesc>
+                            <!--  *************************************  <layoutDesc>  ********************************* -->
+                            <layoutDesc>
+                                <layout columns="1" writtenLines="4">
+                                    <p/>
+                                </layout>
+                            </layoutDesc>
+                        </objectDesc>
+
+                        <!--  *************************************  <handDesc>  ********************************* -->
+
+                        <handDesc hands="1">
+                            <handNote ana="#impressed.inscribed"> </handNote>
+                        </handDesc>
+
+                        <!--  *************************************  <decoDesc>  *********************************     -->
+                        <decoDesc>
+                            <decoNote>
+                                <p/>
+                            </decoNote>
+                        </decoDesc>
+                    </physDesc>
+
+                    <!--  *************************************  <history>  *********************************  -->
+
+                    <history>
+
+                        <summary>
+                            <rs/>  <!-- content can be "fake" or blank -->
+                        </summary>
+
+                        <origin>
+                            <date notBefore="0199" notAfter="0299">2nd to 3rd century CE</date>
+                            <placeName ref="asia.asia_minor">Antioch, Turkey</placeName>
+                        </origin>
+                        <acquisition>
+                            <!-- subsequent locations if inscription was moved -->
+                            <date when="0001">2003</date>
+                            <p>Gift of the Committee for the Excavation of Antioch to Princeton
+                                University </p>
+                        </acquisition>
+
+
+                    </history>
+                </msDesc>
+                <!--  *************************************  <listBibl>  *********************************  -->
+                <listBibl>
+                    <bibl>
+                        <ptr type="rest-of-bibl" target="#"/>
```

Comment:
Something is missing here in the bibl? Or if we don't have something, remove the bibl entry.